app/controllers/posts_controller.rb

```diff
@@ -1,11 +1,12 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.where(author_id: @user)
+    @posts = User.find(params[:user_id]).posts.includes(:comments)
```

Comment:
* Kindly include in your PR description the screenshots of your app's logs before and after the fix as requested in the [project requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=Make%20sure%20that%20the,and%20after%20the%20fix)